Dockerfile

```diff
@@ -88,12 +88,22 @@ RUN apk add --no-cache --virtual .or-build-deps \
         perl-dev \
         readline-dev \
         zlib-dev \
+        gnupg \
     && cd /tmp \
     && curl -fSL https://www.openssl.org/source/openssl-${RESTY_OPENSSL_VERSION}.tar.gz -o openssl-${RESTY_OPENSSL_VERSION}.tar.gz \
+    && curl -fSL https://www.openssl.org/source/openssl-${RESTY_OPENSSL_VERSION}.tar.gz.sha256 -o openssl-${RESTY_OPENSSL_VERSION}.tar.gz.sha256 \
+    && sha256sum openssl-${RESTY_OPENSSL_VERSION}.tar.gz | cut -f1 -d" " | grep - -qf openssl-${RESTY_OPENSSL_VERSION}.tar.gz.sha256 \
     && tar xzf openssl-${RESTY_OPENSSL_VERSION}.tar.gz \
     && curl -fSL https://ftp.pcre.org/pub/pcre/pcre-${RESTY_PCRE_VERSION}.tar.gz -o pcre-${RESTY_PCRE_VERSION}.tar.gz \
+    && curl -fSL https://ftp.pcre.org/pub/pcre/Public-Key -o pcre-Public-Key \
+    && curl -fSL https://ftp.pcre.org/pub/pcre/pcre-${RESTY_PCRE_VERSION}.tar.gz.sig -o pcre-${RESTY_PCRE_VERSION}.tar.gz.sig \
+    && gpg --import pcre-Public-Key \
+    && gpg --verify pcre-${RESTY_PCRE_VERSION}.tar.gz.sig pcre-${RESTY_PCRE_VERSION}.tar.gz \
     && tar xzf pcre-${RESTY_PCRE_VERSION}.tar.gz \
     && curl -fSL https://openresty.org/download/openresty-${RESTY_VERSION}.tar.gz -o openresty-${RESTY_VERSION}.tar.gz \
+    && curl -fSL https://openresty.org/download/openresty-${RESTY_VERSION}.tar.gz.asc -o openresty-${RESTY_VERSION}.tar.gz.asc \
+    && gpg --keyserver keys.gnupg.net --recv-key A0E98066 \
```

Comment:
Noticed during local docker image build pgp verification fails for two keys you should use the keyserver to import them i.e. 
gpg --keyserver keys.gnupg.net --recv-key 9766E084FB0F43D8 B550E09EA0E98066
------------ output ----
gpg: key B550E09EA0E98066: public key "Yichun Zhang (agentzh) <agentzh@gmail.com>" imported
gpg: key 9766E084FB0F43D8: public key "Philip Hazel <ph10@hermes.cam.ac.uk>" imported

Reply:
Thanks, changed.